### ISSUE TITLE
Question regarding country_authorship_classifier.py - balancing classes (country authors) across training, validation and test datasets

### ISSUE THREAD

**Issue by ntrinkle** [BryceStansfield/AntarcticResearch#2]
Hi Bryce,

I noticed that so far in your country authorship classifier model, you are testing with a subset of countries (["Australia", "United Kingdom", "United States", "Norway", "Chile"]). 

Have you tested with different numbers of, or combinations of countries? If so, were there any issues with balancing classes (countries) across the train, validation, test sets (ie, so we don't get a training dataset that excludes one of the countries we want to predict authorship for, say if they have contributed a small number of documents)? (I understand you are using multi-label classification, so for a document, binary classification is used to predict each country as an author separately (there may be multiple authors).

Also, did you notice any particular changes in performance or bottlenecks (ie. I noticed that as I increased the number of countries, overall performance (binary cross entropy) improved, but the search for XGBoost hyperparameters takes longer)?

Thanks,
Natasha
